packages/synapse-react-client/src/utils/functions/SqlFunctions.ts

```diff
@@ -27,93 +28,112 @@ export const getIgnoredQueryFilterSearchParamKey = (
+    QUERY_FILTERS_LOCAL_STORAGE_KEY,
+  )
+  let additionalFilters: QueryFilter[] = []
+  if (localStorageQueryFiltersString) {
+    additionalFilters = JSON.parse(
+      localStorageQueryFiltersString,
+    ) as QueryFilter[]
+    // delay clearing out the value so re-rendering during React Strict mode does not destroy the filter
+    setTimeout(
+      () => localStorage.removeItem(QUERY_FILTERS_LOCAL_STORAGE_KEY),
+      1000,
+    )
   }
+  if (searchParams) {
+    const isQueryWrapperKey = (key: string) =>
+      key.startsWith('QueryWrapper') || key.startsWith('__')
+    const searchParamKeys = Object.keys(searchParams)
+    if (
+      searchParamKeys.length === 0 ||
+      searchParamKeys.every(isQueryWrapperKey)
+    ) {
+      return
+    }
 
-  return Object.keys(searchParams)
-    .filter(key => !isQueryWrapperKey(key))
-    .map(key => {
-      switch (operator) {
-        case ColumnSingleValueFilterOperator.EQUAL: {
-          const filter: ColumnSingleValueQueryFilter = {
-            concreteType:
-              'org.sagebionetworks.repo.model.table.ColumnSingleValueQueryFilter',
-            columnName: key,
-            operator: operator,
-            values: [searchParams[key]],
-          }
-          return filter
-        }
-        case ColumnSingleValueFilterOperator.IN: {
-          const filter: ColumnSingleValueQueryFilter = {
-            concreteType:
-              'org.sagebionetworks.repo.model.table.ColumnSingleValueQueryFilter',
-            columnName: key,
-            operator: operator,
-            values: searchParams[key].split(','),
-          }
-          return filter
-        }
-        case ColumnMultiValueFunction.HAS: {
-          const filter: ColumnMultiValueFunctionQueryFilter = {
-            concreteType:
-              'org.sagebionetworks.repo.model.table.ColumnMultiValueFunctionQueryFilter',
-            columnName: key,
-            function: operator,
-            values: searchParams[key].split(','),
-          }
-          return filter
-        }
-        case ColumnSingleValueFilterOperator.LIKE: {
-          // If we use a LIKE statement with a synId the backend will look for a string with the first three
-          // characters being 'syn', however, it stores synIds without 'syn', so the query will fail
-          // The backend usually parses 'syn' out, but not with the LIKE clause since its expecting a regex, so we
-          // parse this out. This will cause a bug if something matches the synId regex but is in free text.
-          const filter: ColumnSingleValueQueryFilter = {
-            concreteType:
-              'org.sagebionetworks.repo.model.table.ColumnSingleValueQueryFilter',
-            columnName: key,
-            operator: operator,
-            // Add wildcards around the value
-            values: [`%${removePrefixIfSynId(searchParams[key])}%`],
+    additionalFilters = additionalFilters.concat(
+      Object.keys(searchParams)
```

Comment:
I think you can just get rid of L61-L66 if you do this:

```suggestion
      Object.keys(searchParams || {})
```

Reply:
Changed the existing test (that has no search params) to provide an empty search params object instead, which failed before this change was made.